### ISSUE TITLE
Travis CI is failing - ERROR: 'molecule/*/molecule.yml' glob failed.  Exiting.

### ISSUE THREAD

**Issue by robled** [oasis-roles/meta_skeleton#30]
Seems to happen consistently on recent runs:

- https://travis-ci.com/oasis-roles/meta_skeleton/jobs/281086151
- https://travis-ci.com/oasis-roles/meta_skeleton/jobs/276446571

**Comment by greg-hellings** [comment]
Yes, this is known. Because we're templating the molecule.yml files in the template but not the .travis.yml file, this is going to happen.

We have a few different possible solutions, but none of them have ever been important enough to tackle:

* Rename .travis.yml to .travis.yml.j2 in this repository. It will avoid the Travis runs in this repository that won't work anyway. Of course, then we have no testing on this repo (we kinda don't right now, so we'd really just be removing the noise around failed results). We talked about doing this and then adding in a .travis.yml file to test this skeleton, but that runs afoul of name collisions from the Ansible templating system which has undefined behavior in that case.
* Use a different CI system to run the tests on this directory. Ideally it would involve actually templating this role out and then running `molecule lint -s foo` on the resulting realized minimal role. The difficulty here is that resulting files would end up in the resulting role directories and could cause confusion (e.g. if we used Github Actions or Circle CI).
* Write something external to this role to test it. Like a custom Jenkins job. The difficulty there is that any test definitions would then live external to this repository and would be more difficult to maintain and manage.

None of those are terrible, they just all have drawbacks.
